### ISSUE TITLE
Scala Packages please?

### ISSUE THREAD

**Issue by Witty-Kitty** [jupyter/docker-stacks#362]
Saddle
JFreeChart
Brunel
Latest versions please.

**Comment by parente** [comment]
Thanks for the suggestion @Witty-Kitty. I'm not a scala packaging expert, but I'm happy to take a pull request adding these from you or anyone else looking to contribute.

**Comment by parente** [comment]
In the year since this was opened, we've started to encourage fewer additions to the images here and more contributions to recipes for user image builds and community maintained stacks. 
We've moved the recipes from the GitHub wiki to a ReadTheDocs site (http://jupyter-docker-stacks.readthedocs.io/en/latest/using/recipes.html), and added information about how to contribute new recipes to the site (http://jupyter-docker-stacks.readthedocs.io/en/latest/contributing/recipes.html). 

I think a recipe about how to add one or more of these packages, or Scala packages in general, to a user-built image would be a nice addition to the docs. I'm tagging this image as such in hopes a  member of the community knowledgable about Scala packaging might work on it.
